rbkit-lib/subscriber.cpp

```diff
@@ -55,6 +55,12 @@ void Subscriber::triggerGc() {
     commandSocket->sendCommand(triggerGC_Command);
 }
 
+void Subscriber::startCPUProfiling() {
+    RBKit::CmdStartProfile startCPUProfile;
```

Comment:
the command used here is wrong. it sends `start_memory_profile` command to backend.